detox/test/integration/utils/simplistic-runner.js

```diff
@@ -11,11 +12,16 @@ async function main() {
 }
 
 async function test1() {
+  log.info('Test 1');
   await device.launchApp();
 }
 
 async function test2() {
+  log.info('Test 2');
   await device.terminateApp();
 }
 
-main().catch((e) => console.error(e));
+main().catch((e) => {
+  console.error(e);
+  process.exit(1);
```

Comment:
Adding a non-zero exit code logic makes this test much stricter and useful. This helped me to find that the tests inside this file were actually crashing. That's how I fixed `!== null` check.